docs/edu/classes.md

```diff
@@ -0,0 +1,33 @@
+# Teaching classes using LEAP-Pangeo
+
+## Can I use LEAP-Pangeo for my class?
+
+LEAP provides infrastructure support to [LEAP-Affiliated Courses](https://leap.columbia.edu/education/education-curriculum/).
+
+If you are an instructor of a LEAP-Affiliated Course and want your students to access LEAP-Pangeo, you should contact the [Data & Compute Team](../../support/contact) at least 3 weeks before the class start date in order to coordinate efforts for onboarding.
+
+## Preparing to teach
+
+### Class Material
+
+We suggest that instructors and teaching assistants preprare class material directly on the hub as much as possible to ensure an identical experience for the students during the class.
+
+To develop, edit, and test class material, instructors and teaching assistants should themselves apply for [membership](https://forms.gle/RpeaMZh5btTdZtzu8) under the **education** category and make sure to check out our **LINK TO GETTING STARTED**.
+
+### Sign up students
+
+All students should apply for [membership](https://forms.gle/RpeaMZh5btTdZtzu8) under the **education** category at least two weeks before the starting date of the class.
+
+```{important}
+In order to gain access to LEAP-Pangeo, a github account is required. It is free to create one. Please instruct your students to create one before requesting access. 
+```
+
+Instructors can verify that students have access by checking that their github usernames as listed as members of [this github team](https://github.com/orgs/leap-stc/teams/leap-pangeo-base-access). We suggest that instructors direct students [Getting Started](../../introduction/getting_started) page and ask them to test their [access to the Hub](../../compute/hub_access) before the class starts to avoid any technical interruptions.
+
+Make sure to familiarize yourself with the troubleshooting steps below before the start of the class, so that you are prepared to sign up students on the first day if necessary.
```

Comment:
Maybe we shift the tone a tad? Like, *we recommend familiarizing yourself with or..*